backend/benefit/calculator/tests/test_calculator_api.py

```diff
@@ -224,7 +240,8 @@ def test_application_replace_pay_subsidy(handler_api_client, received_applicatio
     assert response.status_code == 200
     received_application.refresh_from_db()
     new_data = HandlerApplicationSerializer(received_application).data
-    del new_data["pay_subsidies"][0]["id"]
+    del new_data["pay_subsidies"][0]["id"]  # id is expected to change
+    del new_data["pay_subsidies"][0]["duration_in_months_rounded"]  # read-only fields
```

Comment:
Why remove the read-only field here? Is it expected to change?

Reply:
duration_in_months_rounded field does not exist in the data that this testcase sends to the server. This testcase does not test the calculation of the duration.